src/test/java/hudson/plugins/openid/OpenIdUserPropertyTest.java

```diff
@@ -39,15 +41,18 @@ public void testRoundtrip() throws Exception {
 
         // submit empty config
         WebClient wc = new WebClient().login("alice", "alice");
-        submit(wc.goTo("user/alice/configure").getFormByName("config"));
+        HtmlPage configure = Jenkins.getVersion().isNewerThan(new VersionNumber("2.467"))
+                ? wc.goTo("user/alice/account/")
+                : wc.goTo("user/alice/configure");
```

Comment:
Since `jenkins.version` is set to `2.479.3`, the version number comparison will always evaluate to true.  It is safe to simplify it
```suggestion
        HtmlPage configure = wc.goTo("user/alice/account/");
```

Reply:
@MarkEWaite done